### DISCUSSION TITLE
What is the bot-list used for?

### DISCUSSION THREAD

**Discussion by laughedelic** [intuit/auto#1960] [Q&A] [answered]
I saw the [bot-list package](https://github.com/intuit/auto/blob/main/packages/bot-list/src/index.ts) and I'm wondering what it is used for. 

I searched the code, but got confused: I see there is some filtering for changelog, but then I also see releases like [this one](https://github.com/intuit/auto/releases/tag/v10.21.2) which mention `@dependabot-preview[bot]`.

**Comment by hipstersmoothie** [comment, marked as answer]
bot-list is just common accounts we don't want included in the changelog (like dependabot) and other parts of auto. If you see a bot missing feel free to make a PR and add one.

Why is that account there? GitHub or dependabot keeps changing how the bot is displayed so it's a game of cat and mouse

**Reply by laughedelic** [reply]
I was thinking of adding `mergify[bot]` (associated with @mergifyio, but it's a different account), but doubted if it belongs to this list. 

I actually want to see Dependabot's PRs in the release notes. Aren't dependency updates valuable for the changelog?

**Reply by hipstersmoothie** [reply]
from a library point of view (IMO) no, but I could see how from an app perspective it might be more userful. In our case it produced a wall of dep updates in the changelog which drowned out the real updates

**Reply by laughedelic** [reply]
I see, different languages, different perspective. I can see JS dependencies being updated every minute 😄 

Since libraries on the JVM are distributed as compiled packages, binary compatibility is a big deal. It happens sometimes that an innocent dependency update introduces a dependency conflict or a binary incompatibility for the consumers. So knowing which dependencies got updated might be important when you look at a library's release notes.

I understand that this list is static at the moment. What do you think about making it configurable? 

It also seems like an overlap with the [omit-release-notes](https://intuit.github.io/auto/docs/generated/omit-release-notes) plugin.

**Reply by hipstersmoothie** [reply]
I'm open to a "disableBotList" option. That plugin is to deal with those dev update tools that sometimes include a "release notes" section that got picked up.

**Reply by hipstersmoothie** [reply]
Or even better a botList option were you can override the default
